packages/plugin-constraints/sources/Constraints.ts

```diff
@@ -164,8 +164,10 @@ export class Constraints {
   constructor(project: Project) {
     this.project = project;
 
-    if (xfs.existsSync(ppath.join(project.cwd, toFilename(`constraints.pro`)))) {
-      this.source = xfs.readFileSync(ppath.join(project.cwd, toFilename(`constraints.pro`)), `utf8`);
+    const constraintsFilename = project.configuration.get(`constraintsFilename`);
+
+    if (xfs.existsSync(ppath.join(project.cwd, toFilename(constraintsFilename)))) {
```

Comment:
Imo the best would be to name it `constraintsPath` and define it as a `SettingsType.ABSOLUTE_PATH`. This would avoid having to make the join, and would be more consistent with the other settings (very few settings should be filename - the reason why we have `lockfileFilename` is that we need the filename it to detect the root of the other path settings).

Reply:
Done 😃